src/modules/ClassStore.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable eqeqeq */
```

Comment:
I know eslint can be pain. However, I think it is still best to follow it and not use disabling comments to avoid the issue.

Reply:
@tkz96 I disabled the `eqeqeq` linting because it was flowing error making comparison statements using double equal signs `==`